src/util.cpp

```diff
@@ -105,9 +105,84 @@ std::string to_internal(const std::string &);
 
 using namespace std;
 
+namespace Logging {
+//Globals defined here because link fails if in globals.cpp.
```

Comment:
space between `//` and the actual comment

Reply:
k